src/todo/UI.lua

```diff
@@ -121,40 +135,47 @@ function todo.create_add_edit_frame(player)
 
     local table = frame.add({
         type = "table",
+        style = "todo_table_default",
         name = "todo_add_task_table",
         column_count = 2
     })
 
     table.add({
         type = "label",
+        style = "todo_label_default",
         name = "todo_add_task_label",
         caption = {"todo.add_task"}
     })
 
-    local textbox = table.add({
+    table.add({
         type = "text-box",
-        name = "todo_new_task_textbox"
+        style = "todo_textbox_default",
+        name = "todo_new_task_textbox",
+        text = task and task.task or ""
```

Comment:
I don't know if this is common in lua, but this is sort of a 'poor man's ternary'.

If task is truthy, then the result of the expression will be task.task. Otherwise, emptystring. Analogous to

text = task ? task.task : ""; in javascript.

(if this is already a well-known pattern, sorry. I'm very new to lua :p)

I used this in a few places, not just here.

Reply:
I'm also new to this :) If it works fine. I think the usage of and and or is extremely unintuitive, though.

Reply:
Yeah, that's the downside of this pattern.  I'm thinking about just putting it in an if block anyways.  This is really nice and concise, but readability is more important.  I'll see what looks better.